src/components/shared/Input.jsx

```diff
@@ -0,0 +1,30 @@
+import useInput from "../hooks/useInput";
+import { Style } from "../styles/Input.styles";
+
+export default function Input({ addTodo }) {
+  const [inputValue, handleChange, setInputValue] = useInput("");
+
+  const handleAddTodo = (event) => {
```

Comment:
이벤트 핸들러의 이름은 `handle` + `대상` + `동작` 의 형식으로 짓습니다!
이 경우, `<form>`에 바인딩 된 `onSubmit` 핸들러이기 때문에 아래와같이 작성할 수 있습니다.

```suggestion
  const handleFormSubmit = (event) => {
```
[참고자료 - [번역] React의 이벤트 핸들러 네이밍 (Event handler naming in react) ](https://blog.sonim1.com/220)